WDE.SmartScriptEditor/SmartData/actions.json

```diff
@@ -4667,6 +4838,163 @@
         "target_is_source": true,
         "implicit_source": false
     },
+    {
+        "id": 131,
+        "name": "SMART_ACTION_SPAWN_SPAWNGROUP",
+        "parameters": [
+            {
+                "type": "Parameter",
+                "name": "GroupID"
+            },
+            {
+                "type": "Parameter",
+                "name": "minDelay"
+            },
+            {
+                "type": "Parameter",
+                "name": "maxDelay"
+            },
+            {
+                "type": "Parameter",
+                "name": "spawnflags"
```

Comment:
actually i just realized...
This SmartActions are only valid for 335 and 434, because master is missing dynamic spawn system

How should we handle this ?